### PR TITLE
Closes 47 create one containerimage for embedding in adwb

### PR DESCRIPTION
**Pull request by jvsoest**

### CODE REVIEW

File: ADWB_container/run.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## build triplifier
+git clone --recurse-submodules https://github.com/MaastrichtU-CDS/triplifier-boot
```
Comment (aiaragomes) [inline ADWB_container/run.sh:2]: Submodule is fetched with a SSH clone, could that cause a problem?

File: ADWB_container/run.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+cd triplifier-boot
+docker run --rm -it \
+    -v $(pwd):/triplifier-boot \
+    -v $(pwd)/../build_triplifier_boot.sh:/triplifier-boot/build.sh \
+    --workdir /triplifier-boot \
+    maven:3.6.0-jdk-11-slim /bin/bash build.sh
+mv target/triplifier-boot-0.0.1-SNAPSHOT.jar ../triplifier-boot-0.0.1-SNAPSHOT.jar
```
Comment (aiaragomes) [inline ADWB_container/run.sh:9]: I need to use sudo to move the jar and also to delete the triplifier-boot dir. I tried to use the `--user` tag for `docker run`, but it didn't help. The output of `docker run` is owned by root. The easy, but not elegant, solution is to add sudo here XD.

**Review by aiaragomes — DISMISSED** [review]
It works for me :). Only two things to consider:
- The `git clone` of the submodule assumes the user has a SSH access key
- In linux, I'm getting ownership issues with the `triplifier-boot` dir, so I had to make a quick fix by adding sudo in a few places, see the last commit

**Comment by shashankmc** [comment]
Mine seems to be getting stuck with data upload. Were you able to proceed further @aiaragomes ?

**Comment by aiaragomes** [comment]
> Mine seems to be getting stuck with data upload. Were you able to proceed further @aiaragomes ?

Same here.

**Review by aiaragomes — APPROVED** [review]

**Pull request merged**